.github/actions/version_updated/check_version_updated.sh

```diff
@@ -5,6 +5,11 @@ do
   echo $changed_file
   if [[ $changed_file == src/tbp/monty/__init__.py ]]; then
     echo "version_updated=true" >> $GITHUB_OUTPUT
+    # grep finds the version line. It will only have two quotes. Then awk uses
+    # the quotes as separators and prints the middle value, which corresponds to
+    # the version inside the quotes.
+    version=$(grep '__version__\s*=\s*' src/tbp/monty/__init__.py | awk -F'"' '{print $2}')
```

Comment:
you could use the tools/print_version command here instead.

Reply:
Yes, I considered it. I believe that in order to use the `print_version` tool, I need to create the conda environment, install the tool, and invoke it. Is that correct?

Reply:
Yep.

Reply:
> Yes. Though you don't have install anything.

What do you mean?

When I run the tool without installing via `python -m tools.print_version.cli -h` I get an error:
```
(tbp.monty) me@not_invented_here tbp.monty % python -m tools.print_version.cli -h 
Traceback (most recent call last):
  File "/opt/anaconda3/envs/tbp.monty/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/anaconda3/envs/tbp.monty/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/me/tbp/tbp.monty/tools/print_version/cli.py", line 15, in <module>
    import semver
ModuleNotFoundError: No module named 'semver'
```

Reply:
I'm going to merge this as is for now and we can revisit in the future.